doc/sphinx/index.latex.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+ The Coq Reference Manual
+==========================
+
+Introduction
+------------
+
+.. include:: introduction.rst
+
+Credits
+-------
+
+.. include:: credits.rst
+
+License
+-------
+
+.. include:: license.rst
+
+The language
+------------
+
+.. toctree::
```

Comment:
No indexes in this table of matters? And any way to share this between the two versions. I'm foreseeing people adding a chapter and forgetting to update the PDF table of matter or moving chapters around but only in one version.

Reply:
OK. Better like this than nothing.

Reply:
Indices are built automatically, and in LaTeX builds (unlike in HTML) you don't have to add them explicitly to the TOC.